app/population/views/edit/edit.njk

```diff
@@ -0,0 +1,40 @@
+{% extends "form-wizard.njk" %}
+
+{% macro display_field_group(group) %}
+  {% set fields = options.fieldGroups[group] %}
+
+  {% set key = "population::daily_view.labels." + group %}
+
+  <div class="govuk-grid-column-full govuk-!-margin-bottom-2">
+    <h2 class="govuk-heading-m">{{ t(key) }}</h2>
+
+    {% for fieldKey in fields %}
+      {% set field = options.fields[fieldKey] %}
+
+      {% if field.component and not field.skip %}
+        {{ callAsMacro(field.component)(field) }}
+      {% endif %}
+    {% endfor %}
+
+    {{ caller() if caller }}
+
+  </div>
+{% endmacro %}
+
+
+{% block fields %}
+
+  <div class="govuk-grid-row">
+    {% call display_field_group('total_space') %}
+    {% endcall %}
+
+    {% call display_field_group('unavailable_space') %}
+    {% endcall %}
+
+    {% call display_field_group('available_space') %}
+    {% endcall %}
+
```

Comment:
```suggestion
```

Reply:
All the various whitespace issues have been tightened up.